swagger.json

```diff
@@ -1821,6 +1821,69 @@
             }
           }
         },
+        "/fleet/vehicles/stats": {
+            "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "name": "startMs",
+                  "description": "Time in Unix epoch milliseconds for the start of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                {
+                  "name": "endMs",
+                  "description": "Time in Unix epoch milliseconds for the end of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                { 
+                  "name": "filter",
+                  "description": "Comma-separated list of stat types. Options are engineState, auxInput1, and auxInput2.",
+                  "type": "string",
+                  "in": "query"
+                }
+              ],
+              "get": {
+                  "tags": ["Fleet"],
+                  "summary": "/fleet/vehicles/stats",
+                  "description": "Fetch engine state and aux input data for all vehicles in the group between a start/end time. The maximum query duration is one hour. Data returned may be affected by device connectivity and processing time.",
+                  "operationId": "GetVehicleStats",
+                  "responses": {
+                    "200": {
+                      "description": "Returns engine state and aux input data for all vehicles in the group between a start/end time, with a maximum query duration of one hour.",
+                      "schema": {
+                        "type": "array",
```

Comment:
Could we return a dictionary at the top level rather than an array? That way the result type is extensible if we want to add additional metadata in the future.

Reply:
Changed the return type